.eslintrc.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
   },
   parserOptions: {
     'sourceType': 'module',
+    'ecmaVersion': 8,
```

Comment:
What  is this doing for us ?  Not a high priority just curios

Reply:
I have added async/await to get all the questions across multiple pages. Using async/await needs ECMAScript version 8 to work.